packages/tasks/src/tasks/index.ts

```diff
@@ -197,6 +198,7 @@ export const TASKS_DATA: Record<PipelineType, TaskData | undefined> = {
 	"any-to-any": getData("any-to-any", placeholder),
 	"audio-classification": getData("audio-classification", audioClassification),
 	"audio-to-audio": getData("audio-to-audio", audioToAudio),
+	"audio-text-to-text": undefined,
```

Comment:
you can put a placeholder here instead

Reply:
Updated here: https://github.com/huggingface/huggingface.js/pull/1033/commits/77a0ea59400a13d91fe518f1b659a146ec49b46f